### DISCUSSION TITLE
How do I remove all the font-related rules?

### DISCUSSION THREAD

**Discussion by abhishek-junghare** [twbs/bootstrap#39843] [Q&A]
I want to exclude everything that is font-related while compiling, such as font-family, font-size, font-weight, etc.

Things that I tried:
1. Excluding _type.scss
2. Removing 'typography' section from variables.scss

**Comment by julien-deramond** [comment]
I don't think you can exclude everything that is font-related, because there are a lot of `font-*` rules defined at different levels of the framework.

However, these `font-*` rules depends on Sass/CSS variables. You could search for them and apply, IDK, a `inherit` value to them so that it can be propagated everywhere.

If you want absolutely no `font-*` rules in your final CSS bundle, maybe applying a post-processing treatment on the file to search and remove these rules could work too.
